src/track/bpm.cpp

```diff
@@ -5,7 +5,7 @@ namespace mixxx {
 /*static*/ constexpr double Bpm::kValueUndefined;
 /*static*/ constexpr double Bpm::kValueMin;
 
-double Bpm::valueFromString(const QString& str, bool* pValid) {
+double mixxx::Bpm::valueFromString(QString str, bool* pValid) {
```

Comment:
why is this not "const QString& str" anymore?

Reply:
Andy why add the Mixxx prefix if it's in the Mixxx name space anyway?